core/build.gradle.kts

```diff
@@ -43,43 +43,39 @@ kotlin {
     tvosSimulatorArm64()
     tvosX64()
 
-    val commonMain by sourceSets.getting
-    val commonTest by sourceSets.getting {
-        dependencies {
-            implementation("org.jetbrains.kotlin:kotlin-test-common")
-            implementation("org.jetbrains.kotlin:kotlin-test-annotations-common")
+    sourceSets {
+        val commonMain by getting
+        val commonTest by getting {
+            dependencies {
+                implementation(kotlin("test"))
+            }
         }
-    }
-    val darwinMain by sourceSets.creating {
-        dependsOn(commonMain)
-    }
-    val darwinTest by sourceSets.creating {
-        dependsOn(commonTest)
-    }
-
-    val androidMain by sourceSets.getting {
-        dependencies {
-            implementation("org.jetbrains.kotlin:kotlin-stdlib")
+        val darwinMain by creating {
+            dependsOn(commonMain)
+        }
+        val darwinTest by creating {
+            dependsOn(commonTest)
         }
-    }
 
-    targets.withType<org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget>().all {
-        val mainSourceSet = compilations.getByName("main").defaultSourceSet
-        val testSourceSet = compilations.getByName("test").defaultSourceSet
+        targets.withType<org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget>().all {
+            val mainSourceSet = compilations.getByName("main").defaultSourceSet
+            val testSourceSet = compilations.getByName("test").defaultSourceSet
 
-        mainSourceSet.dependsOn(darwinMain)
-        testSourceSet.dependsOn(darwinTest)
+            mainSourceSet.dependsOn(darwinMain)
+            testSourceSet.dependsOn(darwinTest)
+        }
     }
 }
 
 android {
-    compileSdk = 30
+    namespace = "co.touchlab.crashkios.core"
```

Comment:
Should consider removing "core" to align with bugsnag, but other than visual consistency, I don't see a compelling reason to do it now.

Reply:
Since file package is also `package co.touchlab.crashkios.core`, maybe we should keep `core` to be aligned with that. 

No issues with removing `core` either.

Reply:
I don't want to do it now. I don't expect any problems from users, but since we get nothing out of changing it, I'd leave it for now.